utils/providers.py

```diff
@@ -368,8 +368,12 @@ def clear_provider_by_type(prov_class, validate=True):
         logger.info(' Providers exist, so removing all {} providers'.format(prov_class))
         paginator.results_per_page('100')
         sel.click(paginator.check_all())
-        toolbar.select('Configuration', 'Remove {} Providers from the VMDB'.format(string_name),
-                       invokes_alert=True)
+        if version.current_version() == version.LATEST:
+            toolbar.select('Configuration', 'Remove {} Providers'.format(string_name),
+                    invokes_alert=True)
+        else:
+            toolbar.select('Configuration', 'Remove {} Providers from the VMDB'.format(string_name),
+                    invokes_alert=True)
```

Comment:
you don't have to do an explicit version pick using if here. 
```python
toolbar.select(
    'Configuration', {
        version.LOWEST: 'Remove {} Providers from the VMDB'.format(string_name),
        version.LATEST: 'Remove {} Providers from the VMDB'.format(string_name),
    },
    invokes_alert=True)
```

Toolbar can take dictionaries and considers tehm to be verpicked.